docs/reference/setup/install/docker.asciidoc

```diff
@@ -425,11 +425,17 @@ chmod g+rwx esdatadir
 chgrp 0 esdatadir
 --------------------------------------------
 
-You can also run an {es} container using both a custom UID and GID. Unless you
-bind-mount each of the `config`, `data` and `logs` directories, you must pass
-the command line option `--group-add 0` to `docker run`. This ensures that the user
-under which {es} is running is also a member of the `root` (GID 0) group inside the
-container.
+You can also run an {es} container using both a custom UID and GID. You
+must ensure that file permissions will not prevent {es} from executing. You
+have two options.
+
+* Pass the `--group-add 0` command line options to `docker run`. This
+  ensures that the user under which {es} is running is also a member of the
+  `root` (GID 0) group inside the container.
+* Alternatively, you must bind-mound the `config`, `data` and `logs`
+  directories. You must also bind-mount the `plugins` directory if you
+  intend to install plugins and prefer not to <<_c_customized_image, create a
+  custom Docker image>>.
```

Comment:
Should we move the bind-mounting bullet first? I imagine most admins are using Docker compose in production so I'm not sure if we want to highlight the `docker run` option first.

Reply:
I'm honestly not sure which option an admin would prefer, but I do suspect that people following the guide to get something up-and-running would be better off using the bind-mounts, so I've swapped the options.

Reply:
>  I imagine most admins are using Docker compose in production so I'm not sure if we want to highlight the docker run option first.

FWIW, I don't think folks using our Docker images in production are using either of these options. They are likely using Kubernetes or some other orchestrator or config as code mechanism to deploy. That's just my intuition though.